de.metas.swat/de.metas.swat.base/src/test/java/de/metas/adempiere/service/impl/PlainCountryDAO.java

```diff
@@ -96,4 +98,29 @@ public boolean accept(final I_C_Country pojo)
 			}
 		});
 	}
+
+	@Override
+	public List<I_C_Country_Sequence> retrieveCountrySequence(I_C_Country country, I_AD_Org org, String language)
```

Comment:
Shall work without this method because Query API is completelly supported on mocked database. Pls remove.